src/index.ts

```diff
@@ -11,9 +11,11 @@ const server = new ApolloServer({
     typeDefs,
     resolvers,
     introspection: true,
+    debug:false,
```

Comment:
This works, but isn't a way we can deduct the stacktrace from the client response but still have it for debugging purposes as this make it completely unavailable.

Reply:
That would be the best compromise in between. We need errors for debugging our code when things break.
We can try based on the `NODE_ENV`

Reply:
> 

Can you try based this or write a formatError function that can log the stacktrace the return the error withour the stacktrace.